docs/azure_jumpstart_ag/contoso_supermarket/freezer_monitor/_index.md

```diff
@@ -4,4 +4,261 @@ weight: 100
+
+- Confirm the dashboard name, accept the suggested name (or choose your own), and click Create.
+
+  ![Confirm dashboard name](./img/adx_confirm_dashboard_report_name.png)
+
+- By default, the simulated IoT sensors are sending data to ADX so you will see at least a few minutes of data in the dashboard. Click Save to save the dashboard in ADX.
+
+  ![Default freezer dashboard](./img/adx_freezer_dashboard_default.png)
+
+## Scenarios
+
+Here are a few scenarios that Contoso Supermarket might encounter, and how they can be addressed using the data collected by the IoT sensors.
+
+### Scenario 1: Identifying the broken freezer
+
+The manager of the Chicago store has reported that food in one of the freezers has not been staying frozen. She has moved the food to the second freezer and called for service, but the technician reported that the freezer seems to be operating within parameters. Your job as the data analyst is to determine if the temperature sensor in the freezer has observed the issue, and provide the data to the store manager.
+
+#### Confirm the issue in Azure Data Explorer
+
+- Open the __Freezer Monitoring__ dashboard in ADX: https://dataexplorer.azure.com/dashboards
+
+  ![Dashboard showing all freezers](./img/adx_freezer_dashboard_default.png)
+
+- The charts are quite busy so let's make it easier to see the problem in Chicago by filtering the dashboard to show only data for the __Chicago__ store.
+
+  ![Dashboard showing filtering for Chicago](./img/adx_freezer_dashboard_select_chicago.png)
+
+- The store manager didn't tell you which freezer was having problems, but if you look at the dashboard below it's obvious that Freezer-2 (aka _device-id:Freezer-2-Chicago:Temperature_F_) is regularly exceeding the safe threshold of 20°F as indicated by the green dashed line. You can send this information to the store manager via a screenshot, or you can show the manager how she can use the Grafana dashboard to see see the same data.
+
+  ![Dashboard showing only Chicago freezers](./img/adx_freezer_dashboard_show_chicago.png)
+
+- While you're viewing the dashboard, take a look at the Seattle store to see if there are any issues that should be reported to that store manager.
+
+#### View the data in Grafana at the store
+
+From the Client VM:
+
+- Open the Edge browser, expand Grafana in the Favorites Bar, and select __Grafana Prod__
+- Login with the username _admin_ and the Windows Administrator password provided when you created the deployment
+- Click the hamburger menu next to __Home__ then click __Dashboards__
+
+  ![Grafana showing the Dashboards menu](./img/grafana_click_dashboards.png)
+- Click __General__ to see the list of dashboards then click __Chicago - Freezer Monitoring__ to open the dashboard for Chicago
+
+  ![Grafana showing list of Dashboards](./img/grafana_click_chicago.png)
+  - Notice that __freezer2__ is showing significant variability and frequently exceeding the safe threshold of 20°F.
```

Comment:
there is an extra tab/space here before the bullet that seems out of place

Reply:
also missing blank line

Reply:
fixed